### PR TITLE
[4.0] Workflow Enable/Disable

### PR DESCRIPTION
**Pull request by brianteeman**
Instead of published/unpublished/trashed this pr changes the status for workflows/stages/transitions to enabled/disabled/trashed

This PR required fixing an obvious bug in the js select-colouur
Note that you will need to do a full npm install to test this

image

Pull Request for Issue #21757

### CODE REVIEW

**Comment by brianteeman** [comment]
@dgrammatiko can you double check the is changes please

File: administrator/language/en-GB/en-GB.com_workflow.ini [hunk 1]
```diff
@@ -98,3 +98,5 @@ COM_WORKFLOW_WORKFLOW_NOTE="Note"
 COM_WORKFLOW_WORKFLOWS_ADD="Add Workflow"
 COM_WORKFLOW_WORKFLOWS_EDIT="Edit Workflow"
 COM_WORKFLOW_WORKFLOWS_LIST="Workflows List"
+JLIB_HTML_PUBLISH_ITEM="Enable"
+JLIB_HTML_UNPUBLISH_ITEM="Disable"
```
Comment (infograf768) [inline administrator/language/en-GB/en-GB.com_workflow.ini:102]: Where are these used?

**Comment by infograf768** [comment]
Also: there are changes to 2 package-lock.json? They do not look like related to this PR.

**Comment by wojsmol** [comment]
@infograf768 package-lock.json is wrongly regenerated - this depends on npm and node version - see https://github.com/joomla/joomla-cms/pull/21772

File: administrator/language/en-GB/en-GB.com_workflow.ini (hunk 1 above)
Comment (brianteeman) [reply administrator/language/en-GB/en-GB.com_workflow.ini:102]: When you hover on the publish/unpublish icon in the list of workflows/states/transitions - its done the same as in the plugin manager

**Comment by brianteeman** [comment]
removed the lock files

**Comment by bembelimen** [comment]
I have tested this item :white_check_mark: successfully on 113da70dada085803afc4ec162c548feda355436<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21836">issues.joomla.org/tracker/joomla-cms/21836</a>.</sub>

**Comment by chmst** [comment]
I have tested this item successfully, the options for status are changed as described.

Toolbar-Buttons, messages and the Table select are still the publish/unpublish.

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 113da70dada085803afc4ec162c548feda355436<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21836">issues.joomla.org/tracker/joomla-cms/21836</a>.</sub>

**Comment by ghost** [comment]
Ready to Commit after two successful tests.

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**